### ISSUE TITLE
Adding StackOverflow

### ISSUE THREAD

**Issue by mmahmoudian** [bilelmoussaoui/revoke.app#8]
In the URL structure of StackOverflow, the userID is an integral part and you cannot remove it. For example, the URL to access my Settings > Applications is:
`https://stackoverflow.com/users/apps/1613005`
which the last part is my userID.

I couldn't find a universal way to address this.

**Comment by bilelmoussaoui** [comment]
Hmm that's a hard one to do. I don't think we have any easy way to do this one :(

**Comment by mmahmoudian** [comment]
Perhaps one way would be to open a modal window and explain to user how to find the page. Although I don't know how that can be defined in a .toml file!

There are many websites like this and we should find a general solution for this.

**Comment by bilelmoussaoui** [comment]
yeah, we will figure something out:) the format of the toml file could grow into something that handles such case as well
